go/vt/wrangler/tablet_test.go

```diff
@@ -48,20 +49,14 @@ func TestInitTabletShardConversion(t *testing.T) {
 		Shard:    "80-C0",
 	}
 
-	if err := wr.TopoServer().InitTablet(context.Background(), tablet, false /*allowPrimaryOverride*/, true /*createShardAndKeyspace*/, false /*allowUpdate*/); err != nil {
-		t.Fatalf("InitTablet failed: %v", err)
-	}
+	err := wr.TopoServer().InitTablet(context.Background(), tablet, false /*allowPrimaryOverride*/, true /*createShardAndKeyspace*/, false /*allowUpdate*/)
+	require.NoError(t, err, "InitTablet failed")
```

Comment:
No need for the last string here, just `require.NoError(t, err)` suffices. That applies to other cases as well.

Reply:
@Ari1009 This comment applied to all the other cases too. There's many cases where it's not useful / needed to add the additional logic.